include/seastar/core/internal/io_desc.hh

```diff
@@ -30,6 +30,7 @@ class kernel_completion {
 protected:
      kernel_completion() = default;
      kernel_completion(const kernel_completion&) = default;
+     kernel_completion& operator=(const kernel_completion&) = default;
```

Comment:
completions should be move-able. Won't compiler auto-delete the move constructor for it after it finds the user-defined copy-constructor?

Reply:
ahh, i think so. fixed.